src/pages/CoinDetailPage.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { fetchSingleCoinAction } from '../redux/slices/singleCoinSlice';
+import '../styles/CoinDetailPage.css';
+
+const CoinDetailPage = () => {
+  const { coinId } = useParams();
+  const dispatch = useDispatch();
+  const coinData = useSelector((state) => state.singleCoin.coindata) || {};
+
+  useEffect(() => {
+    dispatch(fetchSingleCoinAction(coinId));
+  }, [dispatch, coinId]);
+
+  return (
+    <div className="coin-detail-page">
+      <div className="coin-header">
+        <img
+          className="coin-image"
+          src={coinData.image}
+          alt={`${coinData.name} Logo`}
+        />
+        <div className="coin-name-price">
+          <h2>{coinData.name}</h2>
+          <h3 className="coin-price">
+            Current Price: $
+            {coinData.price}
+          </h3>
+        </div>
+      </div>
+      <div className="details-header">
+        <h1>
+          Explore
+          {coinData.name}
+          {' '}
+          Details
+        </h1>
+        <p>
+          Get comprehensive information about ⭐
+          {coinData.name}
+          {' '}
+          performance
+          below👇🏾
+        </p>
+      </div>
+
+      <div className="coin-details">
+        <div className="singleCoinDetails">
+          <h3>Market Cap Rank</h3>
+          <p>{coinData.market_cap_rank}</p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Market Cap</h3>
+          <p>
+            $
+            {coinData.market_cap}
+          </p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>24h High</h3>
+          <p>
+            $
+            {coinData.high_24h}
+          </p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>24h Low</h3>
+          <p>
+            $
+            {coinData.low_24h}
+          </p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Price Change 24h</h3>
+          <p>
+            $
+            {coinData.price_change}
+          </p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Market Cap Change</h3>
+          <p>
+            $
+            {coinData.market_cap_change}
+          </p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Total Supply</h3>
+          <p>{coinData.total_supply}</p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Max Supply</h3>
+          <p>{coinData.max_supply}</p>
+        </div>
+        <div className="singleCoinDetails">
+          <h3>Circulating Supply</h3>
+          <p>{coinData.circulating_supply}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
```

Comment:
I can see you have made some changes here. Kudos, but there are still some requested changes which are yet to be implemented. 

- [ ] Kindly try as much as possible to use the same font color throughout to match the actual design. There seems to be two different colors now (black and white) which deviates from the purpose of this project. :+1: 
- [ ] You also need to align each detail content correctly as seen in the main design so that it will be in a straight line or row instead of a column
- [ ] Also, kindly consider updating the background colors since they are not the same in the main design as shown below. It seems to alternate between dark and lighter versions of the background color. This will make your project more beautiful and conscise to the design. :wink: 
- [ ] [OPTIONAL] Your design is missing the white arrows to the far end of each detail card. Kindly consider adding them in order to match the actual design 


### Your design 

![Screenshot from 2023-08-26 16-32-42](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/65233860/b110ac78-67dc-41f9-bc0c-3366ef48de81)



### Actual design 

![Screenshot from 2023-08-26 16-32-24](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/65233860/2532b542-40bb-4052-9e38-c6ef89bf7505)